test/tools/OpenCover/OpenCover.psm1

```diff
@@ -519,6 +520,9 @@ function Invoke-OpenCover
             # poll for process exit every 60 seconds
             # timeout of 6 hours
             $timeOut = ([datetime]::Now).AddHours(6)
```

Comment:
It typically takes around 2.5 - 3 hours total, out of which maybe 2 hours for the unelevated one. '6 hours' was completely pulled out of thin air, to be substantially large enough.